nmdc_server/api.py

```diff
@@ -383,14 +383,19 @@ async def ping_celery(token: Token = Depends(admin_required)) -> bool:
     tags=["jobs"],
     responses=login_required_responses,
 )
-async def run_ingest(token: Token = Depends(admin_required), db: Session = Depends(get_db)):
+async def run_ingest(
+    token: Token = Depends(admin_required),
+    db: Session = Depends(get_db),
+    function_limit: int = None,
+    skip_annotation: bool = False,
```

Comment:
For consistency, let's use a JSON-encoded body for POST request params. (This will mean adding a new schema in the `schemas` module and setting `response_model=...` in the decorator above.)

Reply:
This is the only function intended to be used by non-Kitware folks through the swagger API, so using query params means that there's UI to enter just a number and "true/false" dropdown.  JSON api will make this harder to use.

I'll change it if you think it's important, but I think this way is more user-friendly.